test/AllInOne.cs

```diff
@@ -841,8 +830,27 @@ string LocalFunction(string arg)
             }
         }
     }
+
+    class CSharp7Features
+    {
+        (string, string, long) Tuples(long id)
+        {
+            int a;
+            // variable assignment is valid inside a tuple literal.
+            var foo = (2, a=1);
+
+            var unnamed = ("one", "two", "three");
+            var named = (first: "one", second: "two");
+            (string, long) foo = ("foo", 42);
+            (int? a, int? b) nullableMembers = (5, 10);
+            (int, (int, int)) nestedTuple = (1, (2, 3));
+            var (a, b, c) = Tuples(42);
+            (a, b, c) = (a, b, c);
+            return (named.first, named.second, id);
+        }
+    }
 }
 #line 6
 #line 2 "test.cs"
 #line default
-#line hidden
\ No newline at end of file
+#line hidden
```

Comment:
Technically, the absence of final newline in the original file ensured that a newline was inserted by Prettier. However, I realize that this is pointless to test that as this is dealt with and tested by Prettier. No need to test it again here.